bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBFToXToHH_UL/VBFToBulkGravitonToHH_M/VBFToBulkGravitonToHH_M_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model RS_bulk_ktilda
+define q = u c d s b u~ c~ d~ s~ b~
+define p = u c d s b u~ c~ d~ s~ b~ 
```

Comment:
all models are 4 fs. you cannot add the massive b to the proton. also maxjetflavor and asrwgtflavor are set to 4. 
if you don't want to use 5fs, please remove the b's.